AsyncDisplayKit/ASDisplayNode+Beta.h

```diff
@@ -147,4 +148,38 @@ typedef struct {
 
 @end
 
+#pragma mark - Yoga Layout Support
+
+#if YOGA
+
+@interface ASDisplayNode (Yoga)
+
+@property (nonatomic, strong) NSArray *yogaChildren;
+
+- (void)addYogaChild:(ASDisplayNode *)child;
+- (void)removeYogaChild:(ASDisplayNode *)child;
+
+// This method should not normally be called directly.
+- (ASLayout *)calculateLayoutFromYogaRoot:(ASSizeRange)rootConstrainedSize;
+
+@end
+
+@interface ASLayoutElementStyle (Yoga)
+
+@property (nonatomic, assign, readwrite) ASStackLayoutDirection direction;
```

Comment:
@maicki I decided to put these on the style object directly for a few reasons:

- They can be nicely hidden in +Beta
- The impact in the .mm is very small
- They are able to use std::atomic, which is great for perf but also crucial for thread safety (extensibility API is new, and doesn't yet have a solution for locking)

I'm pretty confident these changes won't add any work during refactoring of the layout system, as they are tucked into the style object without business logic integrations.